pkg/scheduler/framework/cycle_state.go

```diff
@@ -61,15 +61,15 @@ func NewCycleState() *CycleState {
 	return &CycleState{}
 }
 
-// ShouldRecordPluginMetrics returns whether PluginExecutionDuration metrics should be recorded.
+// ShouldRecordPluginMetrics returns whether metrics.PluginExecutionDuration metrics should be recorded.
 func (c *CycleState) ShouldRecordPluginMetrics() bool {
 	if c == nil {
 		return false
 	}
 	return c.recordPluginMetrics
 }
 
-// SetRecordPluginMetrics sets recordPluginMetrics to the given value.
+// SetRecordPluginMetrics toggles the value returned by ShouldRecordPluginMetrics.
```

Comment:
The original comment sounds more reasonable to me, any concern here?

Reply:
redirecting the set to the get centralizes documentation, vs relying on copy/paste the same thing.

Reply:
reverted